api/controllers/deploy.go

```diff
@@ -78,3 +79,8 @@ func (d *DeployController) ListDeploys(c *fireball.Context) (fireball.Response,
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (d *DeployController) UpdateDeploy(c *fireball.Context) (fireball.Response, error) {
```

Comment:
There is no `UpdateDeploy`

Reply:
I have removed the implementation, including the route definition at the top of the file